venvctrl/venv/base.py

```diff
@@ -153,9 +153,14 @@ def shebang(self):
         with open(self.path, 'rb') as file_handle:
             hashtag = file_handle.read(2)
             if hashtag == b'#!':
+                # Check if we're using the new style shebang
+                new_style_shebang = \
+                    file_handle.readline() == b'/bin/sh\n' and \
+                    file_handle.read(8) == b"'''exec'"
 
                 file_handle.seek(0)
-                return file_handle.readline().decode('utf8')
+                return [next(file_handle).decode('utf8') for _
```

Comment:
Is there a way we can handle this while still returning a string? For example, could we return this with a `.join(os.linesep)` so that the exec-style shebang still comes out as a string? 

My concern is that this is a breaking change in the API due to the data type changing. While I believe most users of this use the CLI rather than the code interface, I'd still prefer to find a way that doesn't break the code API if possible.

Reply:
@kevinconway made the changes requested, it feels slightly clunkier to join/split it, but it should maintain API compatibility now.

Reply:
I agree that it feels clunkier but I do appreciate you making the changes. The older a library gets the more I worry that small changes can break builds so I end up being very conservative with API changes.